Assets/Plugins/Editor/Uplift/DependencyResolution/PackageListStub.cs

```diff
@@ -0,0 +1,87 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Uplift.Common;
+using Uplift.Schemas;
+using Uplift.Packages;
+using System.Linq;
+
+namespace Uplift.DependencyResolution
+{
+	class PackageListStub : PackageList
+	{
+		Dictionary<string, Upset[]> dicoPackages;
+
+		public PackageListStub(Dictionary<string, Upset[]> dicoPackages)
+		{
+			this.dicoPackages = dicoPackages;
+		}
+
+		public List<Upset> GetPackages(string requirementName)
+		{
+			List<Upset> listPackages = new List<Upset>();
+			//Check if package is in repo
+			if (dicoPackages[requirementName] != null)
+			{
+				foreach (Upset pkg in dicoPackages[requirementName])
+				{
+					listPackages.Add(pkg);
+				}
+			}
+			return listPackages;
+		}
+
+		public List<PackageRepo> GetAllPackageRepo()
+		{
+			List<PackageRepo> listPackageRepo = new List<PackageRepo>();
+			List<Upset> listUpset = new List<Upset>();
+
+			foreach (Upset[] listValues in dicoPackages.Values)
+			{
+				listUpset.AddRange(listValues);
+			}
+
+			foreach (Upset pkg in listUpset)
+			{
+				PackageRepo correspondingRepo = new PackageRepo();
+				correspondingRepo.Package = pkg;
+				correspondingRepo.Repository = new GithubRepository();
+
+				listPackageRepo.Add(correspondingRepo);
+			}
+			return listPackageRepo;
+		}
+
+
+		override public string ToString()
```

Comment:
Cool

Reply:
Although we could try to remain consistent in our method definitions. See `DependencyGraph` that declares it `public`